tests/models/test_collection.py

```diff
@@ -908,11 +919,18 @@ def remove_work(self, work):
         # has any LicensePools), but not the second.
         assert [work] == index.removed
 
-        # The ExternalIntegration and its settings have been deleted.
-        assert integration not in self._db.query(ExternalIntegration).all()
+        # The ExternalIntegration, its settings and links have been deleted.
```

Comment:
There are two `ExternalIntegration`s at play here, so it would be helpful to distinguish between them:
```
# The collection ExternalIntegration, its settings, and links to other integrations have been deleted.
# The storage ExternalIntegration remains.
```

Reply:
I fixed the comment.